warp-core/src/test/scala/com/workday/warp/math/linalg/RobustPcaRunnerSpec.scala

```diff
@@ -54,7 +54,7 @@ class RobustPcaRunnerSpec extends WarpJUnitSpec {
   def testDoubleRobustPca(): Unit = {
     val runner: RobustPcaRunner = RobustPcaRunner(useDoubleRpca = true)
 
-    val dataSize: Int = 1000
+    val dataSize: Int = 200
```

Comment:
What's the reasoning for this? Does it make any difference in test time?

Reply:
yeah, the test takes 4-5 seconds instead of 20-30

Reply:
Oh sweet! Yeah that sounds great to me